### PR TITLE
Factory defs updates

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for e3380c7371b4aedeff392be8e58645c9d90cb8ff are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1407/docs/artifacts/html/index.html

File: source/reference-manual/factory/factory-definition.rst [hunk 1]
```diff
@@ -222,12 +222,37 @@ containers:
...
+----------
+Optionally use a custom version of ci-scripts_ to perform CI builds.
+
+ ci_scripts:
+  url:
+    **Optional:** Git URL to clone
+
+    **Default:** https://github.com/foundriesio/ci-scripts
+  git_ref:
+    **Optional:** Git tag, branch, or SHA to use
```
Comment (Tim-Anderson) [inline source/reference-manual/factory/factory-definition.rst:247]: if I am reading this correctly, this is where you can specify the SHA of the ci-script to run if you want to lock it down, is that correct?

Comment (doanac) [reply source/reference-manual/factory/factory-definition.rst:247]: yes

**Review by kprosise — APPROVED** [review]
LGTM

**Pull request merged**